Module/PSWordCloud.psm1

```diff
@@ -0,0 +1,23 @@
+﻿$PlatformFolder = switch ($true) {
+    $IsWindows {
+        if ([IntPtr]::Size -eq 8) {
```

Comment:
Use `[Environment]::Is64BitProcess` instead, no need for this backwards-compat voodoo 😄

Reply:
I think I can blame @seeminglyscience for this one, oops!

Reply:
Yep that's my bad habit.  I had mixed luck with that property a long time ago, but it should be fine now (assuming there even was a problem with it to begin with 🤷‍♂️)

Reply:
In scripts that need backwards compatibility with earlier versions of Windows it makes since, since it isn't available in the BCL prior to .NET 4.0, but here it's wholly unnecessary :)